examples/nnp_training.py

```diff
@@ -156,6 +156,27 @@
 nn = torchani.ANIModel([H_network, C_network, N_network, O_network])
 print(nn)
 
+###############################################################################
+# Initialize the weights and biases.
+#
+# .. note::
+#   Pytorch default initialization for the weights and biases in linear layers
+#   is Kaiming uniform. See: `TORCH.NN.MODULES.LINEAR`_
+#   We initialize the weights similarly but from the normal distribution.
+#   The biases were initialized to zero.
+#
+# .. _TORCH.NN.MODULES.LINEAR:
+#   https://pytorch.org/docs/stable/_modules/torch/nn/modules/linear.html#Linear
+
+
+def init_params(m):
+    if isinstance(m, torch.nn.Linear):
+        torch.nn.init.kaiming_normal_(m.weight, a=math.sqrt(5))
+        torch.nn.init.zeros_(m.bias)
+
+
+nn.apply(init_params)
```

Comment:
Great! very clean!

Reply:
Great! thanks for comments, I applied the changes!